go/vt/worker/diff_utils.go

```diff
@@ -93,10 +93,10 @@ func orderedColumns(tableDefinition *tabletmanagerdatapb.TableDefinition) []stri
 	return result
 }
 
-// uint64FromKeyspaceID returns a 64 bits hex number as a string
-// (in the form of 0x0123456789abcdef) from the provided keyspaceId
-func uint64FromKeyspaceID(keyspaceID []byte) string {
-	hex := hex.EncodeToString(keyspaceID)
+// uint64FromShardingKey returns a 64 bits hex number as a string
```

Comment:
I don't think this is correct, this is the keyspace_id (as opposed to sharding key, which is the before-hash value).